### PR TITLE
doc(sas_url): document sas url generation

### PR DESCRIPTION
**Pull request by gjolly**
Describe how to generate a SAS URL for an Azure Table.

### CODE REVIEW

File: docs/SAS_URL.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# How to generate a SAS URL to access an Azure table
+
+Once the subscription table created in the storage account, go on the storage account panel on [portal.azure.com](https://portal.azure.com) and click on `Sharedaccess signature`. Fill the form as follow and click `Generate SAS and connection string`:
```
Comment (CodeMouse92) [inline docs/SAS_URL.md]: There are a couple of typos here:

> Once the subscription table **is** created in the storage account...
>
> ...and click on `Shared access signature`...

File: docs/SAS_URL.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (CodeMouse92) [inline docs/SAS_URL.md:4]: Add an instruction that they need to provide us with the Table service SAS URL.

**Review by CodeMouse92 — CHANGES_REQUESTED** [review]

**Comment by CodeMouse92** [comment]
@gjolly Once you take care of my other comment (still pending), I can merge this and push it all to `main`.

File: docs/SAS_URL.md (hunk 2 above)
Comment (gjolly) [reply docs/SAS_URL.md:4]: I already addressed this comment, see line 7

**Pull request merged**